server/api/notification/notification.model.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  Entity,
+  BaseEntity,
+  Column, PrimaryColumn,
+} from 'typeorm';
+
+@Entity()
+export class Notification extends BaseEntity {
+
+  @PrimaryColumn()
+  id!: string;
+
+  @Column({
+    type: 'double precision',
```

Comment:
The order is currently using whole integers, but I left it as `double precision` in case we ever wanted to add an alert that would be displayed at a specific spot in the existing list.  Instead of having to dump the table and rewrite all the sort fields, we could just use something like `1.5`.